src/Eclo.DataAccess/Repositories/Products/ProductCommentRepository.cs

```diff
@@ -0,0 +1,141 @@
+﻿using Dapper;
+using Eclo.Application.Utilities;
+using Eclo.DataAccess.Interfaces.Products;
+using Eclo.DataAccess.ViewModels.Products;
+using Eclo.Domain.Entities.Products;
+
+namespace Eclo.DataAccess.Repositories.Products;
+
+public class ProductCommentRepository : BaseRepository, IProductCommentRepository
+{
+    public async Task<long> CountAsync()
+    {
+        try
+        {
+            await _connection.OpenAsync();
+            string query = "SELECT COUNT(*) FROM product_comments";
+            var result = await _connection.QuerySingleAsync<long>(query);
+
+            return result;
+        }
+        catch
+        {
+            return 0;
+        }
+        finally
+        {
+            await _connection.CloseAsync();
+        }
+    }
+
+    public async Task<int> CreateAsync(ProductComment entity)
+    {
+        try
+        {
+            await _connection.OpenAsync();
+
+            string query = "INSERT INTO public.product_comments(product_id, user_id, reply_comment_id, comment, is_edited, created_at, updated_at) " +
```

Comment:
Here, the line exceeds 121 characters, please recheck all the code in this section

Reply:
issue has been solved